pkg/clients/ovirt/impl.go

```diff
@@ -479,12 +434,32 @@ func (is *InstanceService) getHostsInCluster(clusterId string) (*ovirtsdk.HostSl
 }
 
 // handleAutoPinning updates the VM after creation to set the auto pinning policy configuration.
-func (is *InstanceService) handleAutoPinning(id string, autoPinningPolicy ovirtsdk.AutoPinningPolicy) error {
-	vmService := is.Connection.SystemService().VmsService().VmService(id)
+func (is *ovirtClient) handleAutoPinning(id string, autoPinningPolicy ovirtsdk.AutoPinningPolicy) error {
+	vmService := is.connection.SystemService().VmsService().VmService(id)
 	optimizeCpuSettings := autoPinningPolicy == ovirtsdk.AUTOPINNINGPOLICY_ADJUST
 	_, err := vmService.AutoPinCpuAndNumaNodes().OptimizeCpuSettings(optimizeCpuSettings).Send()
 	if err != nil {
 		return errors.Errorf("failed to set the auto pinning policy on the VM!, %v", err)
 	}
 	return nil
 }
+
+// getEngineVersion will return the engine version we are using
```

Comment:
This comment seems to be not very useful.

Reply:
+1